### PR TITLE
Add 200 ms offset to state reads so users don't need to.

### PR DESCRIPTION
**Pull request by applmak**

### CODE REVIEW

**Comment by applmak** [comment]
Fixes #10 .

File: lib/shared_state.js [hunk 1]
```diff
@@ -52,6 +52,10 @@ export class SharedState {
   // Returns the value of the shared state, according to the specific kind of
   // variable & interpolator.
   get(t) {
+    // Subtract 200 ms, because in typical operation, the reader (the client) is
+    // going to always be ahead of the writer (the server) by 1-2 server ticks
+    // which is 100ms.
+    t -= 200;
```
Comment (jmegq) [inline lib/shared_state.js]: Please least make it a constant - maybe even express in # of ticks?
CLIENT_STATE_LAG_TICKS = 2  // LOL

Comment (applmak) [reply lib/shared_state.js]: Sure.

**Comment by applmak** [comment]
updated. PTAL.

**Pull request merged**